hypothesis-python/docs/details.rst

```diff
@@ -656,6 +656,8 @@ providing extra information and convenient access to config options.
   :ref:`display test and data generation statistics <statistics>`.
 - ``pytest --hypothesis-profile=<profile name>`` can be used to
   :ref:`load a settings profile <settings_profiles>`.
+  ``pytest --hypothesis-verbosity=<level name>`` can be used to 
```

Comment:
There's an extra space character at the end of this line, which the Travis build is complaining about.